recipe/meta.yaml

```diff
@@ -30,6 +29,7 @@ requirements:
     - setuptools_scm_git_archive
     - libtiledb-sql >=0.10
     - libxcrypt                             # [linux]
+    - mysql-devel
```

Comment:
I think this is still problematic. Everything should come from this package only.

Reply:
Yeah, it is like this in the mysqlclient-feedstock, hence my try:

https://github.com/conda-forge/mysqlclient-feedstock/blob/main/recipe/meta.yaml

Reply:
We don't need it you are right.